server/fleet/campaigns.go

```diff
@@ -70,4 +70,5 @@ type QueryCampaignResult struct {
 	QueryID uint          `json:"query_id"`
 	Error   *string       `json:"error,omitempty"`
 	Results []QueryResult `json:"results"`
+	Err     error         `json:"-"`
```

Comment:
Since the new API has a single campaign, the campaign error now becomes the API response error.